inkcut/ui/manifest.enaml

```diff
@@ -129,48 +138,55 @@ enamldef UIManifest(PluginManifest):
 #            command = 'inkcut.ui.open_preferences'
         MenuItem:
             path = '/settings'
-            label = 'Settings'
+            label = QApplication.translate("settings", "Settings")
             before = 'help'
             after = 'file'
         ActionItem:
             path = '/help/about'
-            label = 'About...'
+            label = QApplication.translate("settings", "About")
             command = 'inkcut.ui.open_webpage'
             parameters = {'url':'https://www.codelv.com/projects/inkcut/'}
         ActionItem:
             path = '/help/project'
-            label = 'Docs...'
+            label = QApplication.translate("settings", "Documentation")
             command = 'inkcut.ui.open_webpage'
             parameters = {'url':'https://www.codelv.com/projects/inkcut/docs/'}
         ActionItem:
             path = '/help/report'
-            label = 'Report issue...'
+            label = QApplication.translate("settings", "Report issue")
             command = 'inkcut.ui.open_webpage'
             parameters = {'url':'https://github.com/codelv/inkcut/issues'}
         ActionItem:
             path = '/help/donate'
-            label = 'Donate...'
+            label = QApplication.translate("settings", "Donate")
             command = 'inkcut.ui.open_webpage'
             parameters = {'url':'https://www.codelv.com/projects/inkcut/support/'}
         ActionItem:
             path = '/help/logs'
-            label = 'Logs...'
+            label = QApplication.translate("settings", "Logs")
             command = 'inkcut.ui.open_webpage'
             parameters = {'url':os.path.expanduser('~/.config/inkcut/logs/')}
         MenuItem:
             path = '/settings/theme'
-            label = 'Theme'
+            label = QApplication.translate("settings", "Theme")
+#        Looper:
+#            iterable = available_styles()+['system']
```

Comment:
Remove the `+ ['system']` and this works

Reply:
it's working, I don't know how I missed that, tanks!